.concourse/pipelines/develop-commit.yaml

```diff
@@ -184,5 +186,10 @@ jobs:
       params:
         mode: normal
         alert_type: failed
+    on_error:
+      put: slack-polymesh
+      params:
+        mode: normal
+        alert_type: failed
```

Comment:
So this will notify me on slack (via some channel... or how?) when I've pushed a failing commit (not necessarily on a PR)?

Reply:
This pipeline is specifically for commits to the `develop` branch.  Since PR merges result in commits to this branch this effectively runs builds for merged PRs.  It will not run on commits to other branches (unless a pipeline specifically for that branch is added).

Reply:
Ah! Seems good.